proj/src/main/webapp/script.js

```diff
@@ -117,15 +117,15 @@ function appendMessage(payload){
  */
 
 const CHAT_ID = '-MB0ycAOM8VGIXlev5u8'
-const PATH = '/chat/'+CHAT_ID+'/messages'; // can make this more detailed (for example add user ID)
+const PATH = '/chat/test/'+CHAT_ID+'/messages'; // can make this more detailed (for example add user ID)
```

Comment:
Is this just for testing. Should this be removed? If not, can you add a `TODO:` to remove it later?

Reply:
so at some point, instead of test it would be a tag name, like running or books, /test/ is just what we're using for development. but yeah i added a todo